src/test/java/com/commercetools/helper/mapper/impl/PaymentMapperImplTest.java

```diff
@@ -0,0 +1,181 @@
+package com.commercetools.helper.mapper.impl;
+
+import com.commercetools.Application;
+import com.commercetools.model.CtpPaymentWithCart;
+import com.paypal.api.payments.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.math.BigDecimal;
+
+import static com.commercetools.payment.constants.paypalPlus.PaypalPlusPaymentIntent.SALE;
+import static com.commercetools.payment.constants.paypalPlus.PaypalPlusPaymentMethods.CREDIT_CARD;
+import static com.commercetools.payment.constants.paypalPlus.PaypalPlusPaymentMethods.PAYPAL;
+import static com.commercetools.testUtil.ctpUtil.CtpResourcesUtil.getPaymentWithCart_complexAndDiscount;
+import static com.commercetools.testUtil.ctpUtil.CtpResourcesUtil.getPaymentWithCart_complexWithoutDiscount;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = Application.class)
+public class PaymentMapperImplTest {
```

Comment:
good catch: so far i have not found any project where real custom line items to use it as a mock cart, but i'd like to have one...